HW_4/task_1.py

```diff
@@ -0,0 +1,27 @@
+# Создайте несколько переменных заканчивающихся и не оканчивающихся на «s».
+# Напишите функцию, которая при запуске заменяет содержимое переменных оканчивающихся на s (кроме переменной из одной буквы s) на None.
+# Значения не удаляются, а помещаются в одноимённые переменные без s на конце.
+
+
+def func() -> None:
+    g = globals()
+    change = {}
+    for key, value in g.items():
```

Comment:
Либо можно взять кортеж, и тогда не придется определять словарь `change `
```suggestion
    for key, value in tuple(g.items()):
```